### PR TITLE
Responsive Home page

### PR DESCRIPTION
**Pull request by Bravax**

### CODE REVIEW

File: frontend/src/components/HomePage/HomePage.scss [hunk 1]
```diff
@@ -1,16 +1,44 @@
 .homePageElements {
   display: flex;
+  flex-direction: row;
```
Comment (ItsMatDev) [inline frontend/src/components/HomePage/HomePage.scss]: already in row when using flex by default

File: frontend/src/components/HomePage/HomePage.scss [hunk 2]
```diff
@@ -1,16 +1,44 @@
...
+
+  .storyRules__playButton {
+    margin-top: 5%;
+  }
 }
 
-@media screen and (min-width: 700px) {
-  .storyRules {
-    padding: 0;
+@media screen and (max-width: 700px) {
```
Comment (ItsMatDev) [inline frontend/src/components/HomePage/HomePage.scss]: mobile first

**Review by ItsMatDev — CHANGES_REQUESTED** [review]

File: frontend/src/components/HomePage/HomePage.scss (hunk 1 above)
Comment (Bravax) [reply frontend/src/components/HomePage/HomePage.scss]: Corrected

File: frontend/src/components/HomePage/HomePage.scss (hunk 2 above)
Comment (Bravax) [reply frontend/src/components/HomePage/HomePage.scss]: Modified

**Review by ItsMatDev — APPROVED** [review]

File: frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.scss [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
   color: $secondaryColor;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  width: 35vw;
-  font-size: 1.1rem;
 
   &__playButton {
+    position: absolute;
+    top: 75%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.scss]: Bottom is better

File: frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
+.rulesTittle {
+  display: flex;
+  flex-direction: row;
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.scss]: Dead code

File: frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.scss [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  align-items: stretch;
+  width: 100%;
+  font-size: small;
+  padding: 1vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.scss]: No vw for height

File: frontend/src/components/HomePage/HomePage.scss [hunk 6]
```diff
@@ -1,44 +1,51 @@
...
 
-  .storyRules__playButton {
-    margin-top: 5%;
+  h2 {
+    font-size: medium;
+  }
+
+  &__playButton {
+    position: absolute;
+    top: 75%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/HomePage.scss]: Bottom is better

File: frontend/src/components/HomePage/HomePage.scss [hunk 7]
```diff
@@ -1,44 +1,51 @@
...
+      height: 85vh;
+      margin-left: 2vw;
     }
 
-    .storyRules {
-      width: 95%;
-      font-size: small;
+    &__playButton {
+      position: absolute;
+      top: 75%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/HomePage.scss]: Bottom is better

Comment (ItsMatDev) [reply frontend/src/components/HomePage/HomePage.scss]: fixed

File: frontend/src/components/HomePage/HomePage.scss (hunk 6 above)
Comment (ItsMatDev) [reply frontend/src/components/HomePage/HomePage.scss]: fixed

File: frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.scss (hunk 5 above)
Comment (ItsMatDev) [reply frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.scss]: fixed

File: frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.scss (hunk 4 above)
Comment (ItsMatDev) [reply frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.scss]: removed

File: frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.scss (hunk 3 above)
Comment (ItsMatDev) [reply frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.scss]: fixed

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**